.github/workflows/deploy-docs.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - uses: ./.github/actions/install-nature
       - name: Install Dependencies
         run: |
-          pip install jupyter sphinx nbsphinx sphinx_rtd_theme 'matplotlib<3.3.0' qiskit-terra[visualization]
+          pip install jupyter 'sphinx<4' nbsphinx sphinx_rtd_theme 'matplotlib<3.3.0' qiskit-terra[visualization]
```

Comment:
@manoelmarques I wanted to get this fix out quickly but maybe we should think about if we can remove these hardcoded packages in favor of using `requirements-dev.txt` or similar?

Reply:
Actually in the CI we don't even need to install sphinx in here because at that point it has already been installed from requirements-dev.txt.